DEVELOPERS

```diff
@@ -215,3 +216,28 @@ If in doubt, reach out on IRC or via a "question" Issue on GitHub.
   - no scraping HTML!
 - New code should be covered by unit-tests
 
+
+Creating a Release
+==================
+
+- Install the tools required:
+   - pip install --editable .[build]
+- Update the NEWS:
+   - python -m towncrier --yes --version `python misc/build_helpers/update-version.py --no-tag`
```

Comment:
Maybe you want `python3` here (and below)?

Reply:
okay, will do .. is that common on macos/windows too? (I usually use `pyenv` to set `python` to mean "whatever version I currently want")

Reply:
I was more thinking that it could potentially guard against situations where `python` is pointing at something else -- like the system `python2` interpreter -- in which case, being explicit here helps to ensure at least that you're getting a newer python (but since you're now explicitly recommending doing all of this in a virtualenv, it matters a lot less since, yeah, `python` will point to the virtualenv's python interpreter)